concat-css.php

```diff
@@ -231,6 +235,41 @@ function css_concat_init() {
 
 	$wp_styles = new Page_Optimize_CSS_Concat( $wp_styles );
 	$wp_styles->allow_gzip_compression = ALLOW_GZIP_COMPRESSION;
+
+	// Add a `move-styles` query param to print the styles after the title element
```

Comment:
Adding query params so we can test printing styles earlier along with this PR. We can remove the commit later if desired.